controllers/remote/cluster_cache.go

```diff
@@ -44,7 +46,11 @@ import (
 const (
 	healthCheckPollInterval       = 10 * time.Second
 	healthCheckRequestTimeout     = 5 * time.Second
-	healthCheckUnhealthyThreshold = 3
+	healthCheckUnhealthyThreshold = 10
+)
+
+var (
+	defaultCacheResync = 10 * time.Minute
```

Comment:
Since we don't actually need to worry about non-k8s apis for this purpose, I'm wondering if it would be ok to let this default to the upstream default (10 hours) rather than setting our own default here.

Reply:
Will revert, it's probably fine given that the controller will still reconcile every 10m